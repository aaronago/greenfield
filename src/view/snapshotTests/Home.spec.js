
import {createSnapshot} from './createSnapshot'

import homeData from './actionPayloads/Home.json'

createSnapshot('Home view loads and renders as expected', homeData.payload)
