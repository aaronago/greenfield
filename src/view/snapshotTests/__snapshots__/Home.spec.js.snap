// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[SNAP] Home view displays overlay loader as expected 1`] = `
<div
  className="sc-ifAKCX fVKAHz"
>
  <div
    className="sc-bdVaJa kdDLWk"
  >
    <h1
      className="sc-bwzfXH dKrTaH"
    >
      Loading
    </h1>
  </div>
  <header
    className="sc-bxivhb eNGRYy"
  >
    <img
      alt="logo"
      className="sc-htpNat lkfFGP"
      src="logo.svg"
    />
    <h2>
      Welcome to 
      
    </h2>
  </header>
  <p>
    Head to the 
    <a
      href="/about"
      onClick={[Function]}
      onMouseDown={false}
      onTouchStart={false}
      target={undefined}
    >
      About Page
    </a>
    .
  </p>
</div>
`;

exports[`[SNAP] Home view loads and renders as expected 1`] = `
<div
  className="sc-ifAKCX fVKAHz"
>
  <header
    className="sc-bxivhb eNGRYy"
  >
    <img
      alt="logo"
      className="sc-htpNat lkfFGP"
      src="logo.svg"
    />
    <h2>
      Welcome to 
      React
    </h2>
  </header>
  <p>
    Head to the 
    <a
      href="/about"
      onClick={[Function]}
      onMouseDown={false}
      onTouchStart={false}
      target={undefined}
    >
      About Page
    </a>
    .
  </p>
</div>
`;
